src/pages/cw-storage-plus/containers/item.mdx

```diff
@@ -1,9 +1,75 @@
 # `Item`
 
-## Overview
+An `Item` is a container that stores a single value under a specific key in
+storage.
 
-## Examples
+Merely constructing the `Item` object does not commit anything to storage. If an
+`Item` has never been written to before, it will be empty.
+
+Use `save` to write to an `Item`.
+
+Use `load` to read from the `Item`, producing an error if the `Item` is empty or
+if deserialization fails.
+
+Use `may_load` if you want to explicitly handle the possibility the `Item` is
+empty - this will produce an
+[`StdError`](https://docs.rs/cosmwasm-std/latest/cosmwasm_std/enum.StdError.html)
+if deserialization fails, but produce an `Ok(None)` if it is empty.
+
+More information can be found in the
+[API docs](https://docs.rs/cw-storage-plus/latest/cw_storage_plus/struct.Item.html).
+
+## Usage examples
 
 ### Saving an admin address
 
-### Saving a config structure
+```rust template="storage"
+use cw_storage_plus::Item;
+
+let admin: Item<String> = Item::new("a");
+assert_eq!(admin.may_load(&storage).unwrap(), None);
+
+admin.save(&mut storage, &"some_address".to_string()).unwrap();
+assert_eq!(admin.load(&storage).unwrap(), "some_address");
+```
+
+### Maintaining a config structure
+
+```rust template="storage"
+use cw_storage_plus::Item;
+use serde::{Serialize, Deserialize};
+
+#[derive(PartialEq, Debug, Serialize, Deserialize)]
```

Comment:
would it be more typical to use `cw_serde` here, or is that just for over-the-wire messages?

Reply:
I forgot this macro exists. Good point, I think it's better to use it - seems like a pure benefit.